### ISSUE TITLE
The format of keypoint

### ISSUE THREAD

**Issue by haofanwang** [Garfield-kh/PoseTriplet#4]
Why you have to convert the keypoint from COCO format 16 joints format? Is it more general to keep COCO format?

**Comment by Garfield-kh** [comment]
Hi, thank you for the interest!
The reason I convert to 16 joints is the introducing the augmenting supervision (i.e., random/GAN projection).
While the projection can produce ground truth 2D-3D pose pair, it requires the 2D-3D pose pair contains the same joint definition. 
Therefore, I either need to change the RL humanoid to have same definition as COCO, or I convert the COCO to 16 joints.
To re-implementing in COCO format, 

1. change the 2D source data in COCO format.
2. re-implement the IK or IKNet, or try from 2D to axis-angle (something like 2D->SMPL, but I didn't make it work in my early experiment if training data is very simple and small.)
3. change the humanoid XML file in RL to contains same joint definition as COCO.
4. change the joint setting to COCO in hallucinator. 

Discussion are welcome~

**Comment by haofanwang** [comment]
Thanks for reply. Is there any way to project 16 joints (3d keypoints)  back into 17 points (coco foramt) by interpolation? And could you also provide the order definition of 16 joints?

**Comment by Garfield-kh** [comment]
Hi, you can found my joint definition [this issue](https://github.com/Garfield-kh/PoseTriplet/issues/2#issuecomment-1093874642), which I will upload the cleaned version later.
For interpolation, 
I think you can try add some mark (e.g., ear/eye) on the RL humanoid to get the position from simulator, or try training a retargeting network (16 joints -> 17 joints).

**Comment by lucasjinreal** [comment]
@Garfield-kh HI, does there any work on how to train a retargeting network?

**Comment by Garfield-kh** [comment]
@jinfagang 
This one:  [DeepMotionEditing](https://github.com/DeepMotionEditing/deep-motion-editing)
Or this one: [Houdini 18.5 Quickstart | KineFX Motion Retargeting](https://www.youtube.com/watch?v=B4RT4OLu5E8)
I never try them, so you may need to check more on it.
